src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,18 @@
 import React from 'react';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
-);
+import { PostInfo } from '../PostInfo';
+import { Data } from '../../types';
+
+type Props = {
+  posts: Array<Data>;
+};
+
+export const PostList: React.FC<Props> = ({ posts }) => {
+  const listItems = posts.map(post => <PostInfo post={post} key={post.id} />);
+
+  return (
+    <ul className="PostList">
+      {listItems}
+    </ul>
```

Comment:
just a suggestion (that's simply not a common practice to perform mapping outside of `ul`)
```suggestion
  return (
    <ul className="PostList">
      {posts.map(post => (
        <PostInfo post={post} key={post.id} />
      )}
    </ul>
```